name.abuchen.portfolio/src/name/abuchen/portfolio/money/Values.java

```diff
@@ -373,6 +373,15 @@ public String format(Double percent)
         }
     };
 
+    public static final Values<Double> PercentPlain2 = new Values<Double>("0.00", 0) //$NON-NLS-1$
+    {
+        @Override
+        public String format(Double percent)
+        {
+            return String.format("%,.2f", percent) + "%"; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Why this new format? If there is a good reason for it instead of simply `Percent`, the name “plain” is definitely inappropriate.

Reply:
The newly introduced format is identical to ```Values.Percent2``` therefore I propose to use that.

But I agree that the number of formats in ```Values``` are confusing... the class has aged.

Reply:
No it's not the same! Percent2 do some magic trick with my number and calculates it "* 100". But my number is already a percent number and did not need an additional "* 100". This Values class is dangerous because you think it does some formatting but it also sometimes do calculations. And this calculations are not comment in the method API doc or visible in the method name! This class should be replaced by a new one without doing unexpected magic.

Reply:
I put a deprecated annotation to this class.

Reply:
Similar to the above: You are not alone in this project. You may prefer to store “+10%” as `10`, and I may prefer to store it as `1.1`, but the standard in this project is to use `0.1`, and all the helper functions are aligned with that.